src/components/accession2/view/Accession2View.tsx

```diff
@@ -55,11 +56,12 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
   const [accession, setAccession] = useState<Accession2>();
   const [openEditLocationModal, setOpenEditLocationModal] = useState(false);
   const [openEditStateModal, setOpenEditStateModal] = useState(false);
+  const [openEndDryingReminderModal, setOpenEndDryingReminderModal] = useState(false);
   const [openDeleteAccession, setOpenDeleteAccession] = useState(false);
   const [openWithdrawModal, setOpenWithdrawModal] = useState(false);
-  const [quantityModalOpened, setQuantityModalOpened] = useState(false);
+  const [openQuantityModal, setOpenQuantityModal] = useState(false);
+  const [openCheckInConfirmationModal, setOpenCheckInConfirmationModal] = useState(false);
```

Comment:
renamed these to have 'openXYZ' instead of 'XYZOpened' - to stay consistent with other open modal properties in this same file.